packages/keyv/src/index.js

```diff
@@ -25,7 +25,7 @@ class Keyv extends EventEmitter {
 			this.store.on('error', error => this.emit('error', error));
 		}
 
-		this.store.namespace = this.options.namespace;
+		this.store.namespace = this.options.namespace ? this.options.namespace + ':' : '';
```

Comment:
IMHO `:` it's a separator and should be taken as internal detail, it not should be part of the namespace value.

In fact, you have the method `_getKeyPrefix` for getting namespace plus separator.

Reply:
aight

Reply:
You can change it, makes more sense to make it internal and not part of namespace